setup.sh

```diff
@@ -26,343 +36,288 @@ WARN="${YELLOW}WARNING:$RESET"
 BLANK="      "
 NEXT="${GREEN}NEXT:$RESET"
 
-get_distribution() {
+get_distribution() (
```

Comment:
why change from `{` to `(`? We get a subshell and avoid leaking to parent/caller?